ui/app/components/tools/wrap.hbs

```diff
@@ -40,16 +40,30 @@
     <div class="box is-sideless is-fullwidth is-marginless">
       <NamespaceReminder @mode="perform" @noun="wrap" />
       <MessageError @errorMessage={{this.errorMessage}} />
-      <div class="field">
-        <div class="control">
-          <JsonEditor
-            @title="Data to wrap"
-            @subTitle="json-formatted"
-            @value={{this.wrapData}}
-            @valueUpdated={{this.codemirrorUpdated}}
-          />
-        </div>
-      </div>
+      <Toolbar>
+        <ToolbarFilters>
+          <Toggle @name="json" @checked={{this.showJson}} @onChange={{action "handleToggle"}}>
```

Comment:
using the `action` helper is actually an outdated Ember pattern. Since `tools/wrap.js` is a glimmer component we want to call this function `this.handleToggle` ([docs](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/#actions))